sold.cc

```diff
@@ -524,7 +524,10 @@ void Sold::LoadDynSymtab(ELFBinary* bin, std::vector<Syminfo>& symtab) {
 
         Syminfo* found = NULL;
         for (int i = 0; i < symtab.size(); i++) {
-            if (symtab[i].name == p.name && symtab[i].soname == p.soname && symtab[i].version == p.version) {
+            // TODO (akawashiro): Do not ignore the soname and version.
```

Comment:
https://github.com/pfnet/sold/issues/10

Reply:
I handled this problem by considering the cases where symtab[i] is VER_NDX_LOCAL and p has a specific version
https://github.com/akawashiro/sold/pull/15/files#diff-ea346e7232da277fcbfd75eb0102ceed1fca1e6c180b2bd4f64d40773c83717bR514-R528

but I still don't fully understand the elf structure and whether my workaround was correct or not. So I think the current workaround is okay